01_fib/saurabhvyas9/main_test.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFib(t *testing.T) {
+	testCases := []struct {
+		numInput       int
+		expectedOutput string
+	}{
+		{numInput: 7, expectedOutput: "1 1 2 3 5 8 13 "},
+		{numInput: -7, expectedOutput: "1 -1 2 -3 5 -8 13 "},
+		{numInput: 10, expectedOutput: "1 1 2 3 5 8 13 21 34 55 "},
+		{numInput: -10, expectedOutput: "1 -1 2 -3 5 -8 13 -21 34 -55 "},
+		{numInput: 0, expectedOutput: ""},
+		{numInput: 1, expectedOutput: "1 "},
+	}
+	for _, test := range testCases {
+		var buf bytes.Buffer
+		out = &buf
+		input := test.numInput
+		fib(input)
+		actual := trueOutput(buf.String())
+		expected := test.expectedOutput
+		assert.Equalf(t, expected, actual, "Febonacci series failed.")
+	}
+}
+
+func trueOutput(temp string) string {
```

Comment:
Thanks. I updated the name of func and used ReplaceAll instead of Replace. 
I also tried by not using this function at all and directly using the second option as you mentioned with strings.Join. 
However, as of now, left the final commit with using the ReplaceAll function for now.